Framework/Kernel/inc/MantidKernel/DateAndTime.h

```diff
@@ -114,30 +114,24 @@ class MANTID_KERNEL_DLL DateAndTime {
   static void createVector(const DateAndTime start,
                            const std::vector<double> &seconds,
                            std::vector<DateAndTime> &out);
-  static bool stringIsISO8601(const std::string &str);
+
+  /// The difference in seconds between standard unix and gps epochs.
```

Comment:
Is there a reason we have one defined in the header and the others in the `.cpp` file?

Reply:
Yes, we cannot define non-integral types in the header. Builds fail.

Reply:
@martyngigg I do not think we can initialise non-integral types in a header without some compilers complaining.

Reply:
Yes, static non-integral types have to be defined out of line. I was suggesting that we be consistent and put `EPOCH_DIFF` in `.cpp` too. I'm not overly concerned though.